packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -70,6 +70,23 @@ export const certificateTransactionsWithEpochs = (
       transactions.map((tx) => ({ epoch: slotEpochCalc(tx.blockHeader.slot), tx }))
     )
   );
+
+export const createDelegationPortfolioTracker = (transactions$: Observable<TxWithEpoch[]>) =>
+  transactions$.pipe(
+    map((transactionsWithEpochs) => {
+      const txSorted = transactionsWithEpochs.sort((lhs, rhs) => lhs.epoch - rhs.epoch);
+      const latestDelegation = txSorted.pop();
```

Comment:
Are transactions$ only delegation transactions or all? If all, then probably need to filter them. Otherwise please rename transactions$ or add a doc comment.